pkg/chain/gen/abi/KeepRandomBeaconOperator.go

```diff
@@ -28,7 +28,7 @@ var (
 )
```

Comment:
We will have a conflict with https://github.com/keep-network/keep-core/pull/1708 here and let's use the version from #1708 when resolving. It has all the required functions as well as the fix for `from` caller address.